### PR TITLE
[docs]: studio faqs

### PR DESCRIPTION
**Pull request by isahers1**

### CODE REVIEW

File: docs/docs/cloud/faq/studio.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Another reason your project might fail to start is because your configuration file is defined incorrectly, or you are missing required environment variables. 
+
+## How does interrupt work?
+
+When you select the `Interrupts` dropdown and select a node to interrupt the graph will pause execution before and after (unless the node goes straight to `END`) that node has run. This means that you will be able to both edit the input to the node and the output of the node. This is intended to allow developers more fine-grained control over the behavior of a node and make it easier to observe how the node is behaving. You will not be able to edit the output if the node is the final node in the graph.
+
+## How do I reload the app?
+
+If you would like to reload the app, don't use Command+R as you might normally do. Instead, close and reopen the app for a full refresh.
```
Comment (nfcampos) [inline docs/docs/cloud/faq/studio.md:21]: why?

File: docs/docs/cloud/faq/studio.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+LangGraph Studio requires Docker Desktop version 4.24 or higher. Please make sure you have a version of Docker installed that satisfies that requirement and also make sure you have the Docker Desktop app up and running before trying to use LangGraph Studio. In addition, make sure you have docker-compose updated to version 2.22.0 or higher.
+
+### Configuration or environment issues
+
+Another reason your project might fail to start is because your configuration file is defined incorrectly, or you are missing required environment variables. 
+
+## How does interrupt work?
+
+When you select the `Interrupts` dropdown and select a node to interrupt the graph will pause execution before and after (unless the node goes straight to `END`) that node has run. This means that you will be able to both edit the input to the node and the output of the node. This is intended to allow developers more fine-grained control over the behavior of a node and make it easier to observe how the node is behaving. You will not be able to edit the output if the node is the final node in the graph.
```
Comment (nfcampos) [inline docs/docs/cloud/faq/studio.md:17]: This is incorrect, you can't directly edit the inputs or outputs of a node

File: docs/docs/cloud/faq/studio.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+
+### Rebuilds from configuration or dependency changes
+
+If you edit your graph configuration file (`langgraph.json`) or the dependencies (either `pyproject.toml` or `requirements.txt`) then the entire image will be rebuilt. This will cause the UI to switch away from the graph view and start showing the logs of the new image building process. This can take a minute or two, and once it is done your updated image will be ready to use!
+
+## Why is my graph taking so long to startup?
+
+The LangGraph studio works by making API calls to the LangGraph API, and this API must be rebuilt regularly to keep up with the changes made to it. For this reason, you might notice that occasionally your graphs take a little longer to startup.
```
Comment (nfcampos) [inline docs/docs/cloud/faq/studio.md:38]: I think this is confusing, sounds like it's making calls to the internet?

File: docs/docs/cloud/faq/studio.md (hunk 1 above)
Comment (isahers1) [reply docs/docs/cloud/faq/studio.md:21]: @dqbd I might be misremembering, but this is from my notes from our conversation

File: docs/docs/cloud/faq/studio.md (hunk 3 above)
Comment (isahers1) [reply docs/docs/cloud/faq/studio.md]: tried to rephrase, but not sure it is correct

File: docs/docs/cloud/faq/studio.md (hunk 2 above)
Comment (isahers1) [reply docs/docs/cloud/faq/studio.md]: changed to "state before node has run" and "state after node has run"

File: docs/docs/cloud/faq/studio.md (hunk 1 above)
Comment (vbarda) [reply docs/docs/cloud/faq/studio.md:21]: yea i think i accidentally triggered it once, not sure why this is the case but it bricks the app

File: docs/docs/cloud/faq/studio.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+# Studio FAQs
```
Comment (vbarda) [inline docs/docs/cloud/faq/studio.md:1]: should this be duplicated in https://github.com/langchain-ai/langgraph-studio or just moved there altogether? cc @hwchase17

File: docs/docs/cloud/faq/studio.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+When you select the `Interrupts` dropdown and select a node to interrupt the graph will pause execution before and after (unless the node goes straight to `END`) that node has run. This means that you will be able to both edit the state before the node is ran and the state after the node has ran. This is intended to allow developers more fine-grained control over the behavior of a node and make it easier to observe how the node is behaving. You will not be able to edit the state after the node has ran if the node is the final node in the graph.
+
+## How do I reload the app?
+
+If you would like to reload the app, don't use Command+R as you might normally do. Instead, close and reopen the app for a full refresh.
+
+## How does automatic rebuilding work?
+
+One of the best features of LangGraph Studio is that it automatically rebuilds your image when you change the source code. This allows for a super fast development and testing cycle which makes it easy to iterate on your graph. There are two different ways that LangGraph rebuilds your image: either by editing the image or completely rebuilding it.
```
Comment (vbarda) [inline docs/docs/cloud/faq/studio.md:25]: (nit) i would remove value judgment here: "best" -> "key"

File: docs/docs/cloud/faq/studio.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+
+### Rebuilds from configuration or dependency changes
+
+If you edit your graph configuration file (`langgraph.json`) or the dependencies (either `pyproject.toml` or `requirements.txt`) then the entire image will be rebuilt. This will cause the UI to switch away from the graph view and start showing the logs of the new image building process. This can take a minute or two, and once it is done your updated image will be ready to use!
+
+## Why is my graph taking so long to startup?
+
+The LangGraph Studio interacts with the LangGraph API through API calls. To stay aligned with ongoing updates, the API requires regular rebuilding. As a result, you may occasionally experience slight delays when starting up your graphs.
```
Comment (vbarda) [inline docs/docs/cloud/faq/studio.md:38]: maybe we should rephrase a bit: "The LangGraph Studio interacts with a local LangGraph API server. To stay aligned with ongoing updates, the LangGraph API requires regular rebuilding. As a result, you may occasionally experience slight delays when starting up your project."

File: docs/docs/cloud/faq/studio.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+## Why are extra edges showing up in my graph?
+
+If you don't define your conditional edges carefully, you might notice extra edges appearing in your graph. This is because without proper definition, LangGraph Studio assumes the conditional edge could access all other nodes. In order for this to not be the case, you need to be explicit about how you define conditional edges. There are two ways you can do this:
```
Comment (vbarda) [inline docs/docs/cloud/faq/studio.md:42]: (nit) maybe "explicit about how you define the nodes the conditional edge routes to"?

**Review by vbarda — APPROVED** [review]

**Pull request merged**